app/controllers/application_controller.rb

```diff
@@ -1,37 +1,53 @@
 class ApplicationController < ActionController::API
   include JsonWebToken
 
-  before_action :authenticate_request, :email_verified?
+  before_action :authenticate_request, :is_email_verified?
 
   private
 
-  def restrict_user
-    if (
-         @current_user.roles.first == patient_role ||
-           @current_user.roles.first == doctor_role
-       )
-      render json: {
-               error: {
-                 message: 'Request forbidden.'
-               }
-             },
-             status: :forbidden
-    end
-  end
-
   def admin_role
     Role.find_by(name: 'admin')
   end
 
+  def doctor_role
+    Role.find_by(name: 'doctor')
+  end
+
   def patient_role
     Role.find_by(name: 'patient')
   end
 
-  def doctor_role
-    Role.find_by(name: 'doctor')
+  def is_admin_role?(user)
+    user.roles.first == admin_role ? true : false
```

Comment:
Nitpick: removed the `true` and `false`
```ruby
user.roles.first == admin_role
```

Reply:
Question:
Is it possible for a user to have a multiple roles?

Reply:
It's possible but ideally, a user will only have one role.

Reply:
okay thanks. for now this will do then